SUASSystem/SUASSystem/utils/image_gps_operations.py

```diff
@@ -0,0 +1,27 @@
+from SUASSystem import GCSSettings, inverse_haversine, get_mission_json
+import matplotlib.path
+import numpy
+import math
+
+def get_target_gps_location(image_midpoint, target_midpoint, drone_gps_location):
```

Comment:
All these new functions that you've added should have a python docstring in the format of:
```
"""
Short description whose length per line does not go beyond 72 column 
spaces.

:param a:   Description of argument a.
:param b:   Description of argument b.

:type a:    List/Boolean/String/etc.
:type b:    Set/Double/Float/etc.

:returns:     Some object/nothing.
"""
```

Even though we're not going to be here next year, it's important that we document what we're adding so that it can be usable and built upon. Plus, getting into the habit of writing documentation is good practice. A docstring takes no more than one minute to write, and can be extremely helpful.